meta/src/store/state_machine.rs

```diff
@@ -512,6 +522,10 @@ impl StateMachine {
 
             WriteCommand::AddDataNode(cluster, node) => self.process_add_date_node(cluster, node),
 
+            WriteCommand::AddNodeMetrics(cluster, node_metrics) => {
```

Comment:
rename AddNodeMetrics to ReportNodeMetrics

Reply:
ok.